frontend/src/components/RegisterForm.vue

```diff
@@ -70,13 +70,28 @@
                   type="password"
                 ></b-form-input>
               </b-form-group>
-
               <b-button variant="danger" style="margin-top: 20px" @click="checkInputs()">Crear usuario</b-button>
       </div>
       </div>
       </div>
       </div>
     </b-container>
+    <!--toast-->
+    <b-toast id="my-toast" toaster="b-toaster-top-center" variant="primary" solid>
+      <template #toast-title>
+        <div class="d-flex flex-grow-1 align-items-baseline">
+          <b-img blank blank-color="#ff5555" class="mr-2" width="12" height="12"></b-img>
+          <strong class="mr-auto">Notice!</strong>
+        </div>
+      </template>
+      <b-row style="margin-left:5px">
+      Se ha enviado un correo de confirmación a su cuenta.
+      </b-row>
+      <b-row align-h="center" style="margin-left:5px">
+      <b-button @click="goLogin()">OK</b-button>
```

Comment:
Que el user deba dar a OK no resulta la forma más intuitiva. Otra opción mejor seria que se se declarase el Toast en el component del navbar y tuviera una "vida" de 5 segundos. De esta forma no se perdería cuando hubiese el cambio de Register => Log In. Es una sugerencia.

Reply:
Lo de cambiar el 'OK' te lo compro, lo puedo cambiar a un 'Confirmar' para que se entienda mejor. En cuanto a lo de añadir el toast al navbar no lo veo tan claro, pienso que este toast es más propio de la ventana de registro y crearlo en el navbar no tendría mucho sentido. Si este toast fuese útil en cualquier pestaña si lo haría, pero no es el caso.

Reply:
La cuestión no es el  nombre del botón, en cualquier página web cuando salen notificaciones tendemos a leerlas e ignorarlas hasta que desaparecen, en pocas nos obligan a darle al OK o CONFIRMAR. Además en este caso se queda esperando hasta que no le das a OK, que resulta bastante confuso.
Cuando digo de ponerlo en el navbar es porque es un componente compartido y esa notificación se mostrará mientras tenga vida y luego desaparecerá.  Al ser compartido, se puede llamar al toast desde qualquier componente donde este.
Pero que seguro que hay más soluciones para mantenerla abierta.
Si no te convence, te la apruebo, pero la navegación será ortopedica.